README.md

```diff
@@ -7,25 +7,38 @@
 
 ## Introduction
 
-Fork and clone this repository. Run the test suite to learn what is required of you.
+We're going to create our own `prime?()` method. This lab will challenge you to
+do a little research to see what options are available to create this method
+without using Ruby's libraries.
 
-Do not use any other Ruby library. You must build a method that can verify whether a number is prime. Don't `require 'math'` and just piggyback off their implementation of prime number.
+Do not use any other Ruby library. You must build a method that can verify
+whether a number is prime. Don't `require 'math'` and just piggyback off their
+implementation of prime number.
+
+Fork and clone this repository. Run the test suite to learn what is required of
+you.
 
 ## Define Methods and Controlling Return Values
 
-You'll be defining a method, `prime?()`, that takes in an integer argument and returns a boolean of whether or not that integer is a prime number.
+Define a method, `prime?()`, that takes in an integer argument and returns a
+boolean of whether or not that integer is a prime number.
 
 ## Use Iteration and Looping
 
-A few things to think about:
-
-* What defines an integer as a prime number? Research algorithms for how to determine if a number is prime.
-* How do you create a range of numbers? How do you turn a range into an array so that it can be iterated over?
+Think about what defines an integer as a prime number. Research algorithms for
+how to determine if a number is prime. Use a range of numbers in an array and
+iterate through it.
 
-Think about the efficiency of your algorithm. How many iterations does it do? Look into Benchmarking in Ruby and profile a few different approaches to implementing `prime?()`.
+You can look into Benchmarking in Ruby and profile a few different approaches to
```

Comment:
Even though this is close to the Resources section, we might want to go ahead and link directly when we mention it here.